advanced_functionality/autogluon-tabular/AutoGluon_Tabular_SageMaker.ipynb

```diff
@@ -526,7 +527,10 @@
     "results = predictor.predict(X_test.to_csv(index=False)).splitlines()\n",
     "\n",
     "# Check output\n",
-    "print(Counter(results))"
+    "threshold = 0.5\n",
+    "y_results = np.array(['yes' if float(i) > threshold else 'no' for i in results])\n",
```

Comment:
Here (and also below), it seems you're expecting `results` to be the output of `predict_proba()`, not the output of of `predict()`. 

If you use `results = predict_proba()` instead, then note that in v0.1 it is now outputting DataFrame by default, so you should replace `results` above with `results['yes']`.